LayoutTests/media/video-pause-while-seeking.html

```diff
@@ -50,7 +50,7 @@
                 consoleWrite("");
                 consoleWrite("<br><em>++ Playing the video");
                 consoleWrite("");
-                run("video.play()");
+                run("video.play().catch((error)=>{})");
```

Comment:
If the `play()` promise rejects we presumably won't fire the `play` event, so won't the test continue to be flaky?

Reply:
Wwwell... if you see [the layout test results in the failing bots](https://build.webkit.org/results/Apple-Ventura-Release-AppleSilicon-WK2-Tests/268273@main%20(5054)/media/video-pause-while-seeking-pretty-diff.html), you'll see that what makes the test flaky (errors sometimes because the expected and actual texts don't match) is just the promise error message. This patch (tentatively) gets rid of the promise message, and the rest of the lines would match.
That said, this makes me think that maybe the failing promise isn't the play() one but some other. That's why this is a tentative patch.